build.gradle

```diff
@@ -347,11 +340,8 @@ dependencies {
 
 // https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
 dependencyCheck {
-	// Specifies if the build should be failed if a CVSS score above a specified level is identified.
-	// range of 0-10 fails the build, anything greater and it doesn't fail the build
-	//FIXME
-	failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 0 : 11
```

Comment:
any reason we've removed the build failure if CVSS score is below 11?

Reply:
This has been now handled by HMCTS plugin.